src/lib/components/file-browser/file-list/file-list.component.html

```diff
@@ -142,4 +116,19 @@
         </div>
         <span>{{ _translationPrefix + '.uploadProgress' | translate: _lang:_progress }}</span>
     </div>
+
+    <ng-template let-data #deleteConfirmationDialog>
+        <h2 mat-dialog-title>{{ _translationPrefix + '.confirmDelete' | translate: _lang }}</h2>
+        <mat-dialog-content>
+            {{ _translationPrefix + '.confirmDeleteMany' | translate: _lang:{ count: _deleteCount } }}
```

Comment:
Es wäre performanter wenn du den deleteCount beim öffnen des Dialogs als data übergeben würdest und dann hier wie folgt darauf zugreifst:
```suggestion
    <ng-template let-deleteCount #deleteConfirmationDialog>
        <h2 mat-dialog-title>{{ _translationPrefix + '.confirmDelete' | translate: _lang }}</h2>
        <mat-dialog-content>
            {{ _translationPrefix + '.confirmDeleteMany' | translate: _lang:{ count: deleteCount } }}
```